### PR TITLE
Color Value Fix

### LINKED ISSUE #20: Background-color value for label is missing "#"
Hi, I have Pimcore v5.6.3

Background-color and border value in style for label is missing "#" so its ignored. Html code for label looks like this:
`<div class="pimcore-workflow-place-indicator" style="background-color: 53c5cc; color:#ffffff; border: 1px solid 53c5cc;" title="New Title">
            New Label
        </div>`

### PR DESCRIPTION
**Pull request by ivanmerta**
Fix color value to have "#" in YAML file. Resolves #20.

### CODE REVIEW

**Comment by dpfaffenbauer** [comment]
please fix the code-style

**Review by paulovitorbal — COMMENTED** [review]
I would like to see both solutions (#23) on code. But it should be tested together.

**Review by dpfaffenbauer — CHANGES_REQUESTED** [review]
code style

**Comment by dpfaffenbauer** [comment]
@paulovitorbal ok, but this PR needs changes then

**Comment by jorisros** [comment]
@paulovitorbal and @ivanmerta I have been tested them both, if the stying is fixed it can merged. I didn't know that we where working both on this issue. (but glad to see community input)

File: src/WorkflowGui/Controller/WorkflowController.php [hunk 1]
```diff
@@ -200,6 +200,11 @@ protected function sanitizeConfiguration($configuration)
     {
         if (isset($configuration['places'])) {
             foreach ($configuration['places'] as $placeKey => &$placeConfig) {
+                if (isset($placeConfig['color'])) {
+                    if (substr($placeConfig['color'], 0, 1) !== '#') {
+					    $placeConfig['color'] = '#'.$placeConfig['color'];
```
Comment (dpfaffenbauer) [inline src/WorkflowGui/Controller/WorkflowController.php]: still not good...

**Comment by ivanmerta** [comment]
Sorry, I did not notice this was being worked on. Thanks.

File: src/WorkflowGui/Controller/WorkflowController.php (hunk 1 above)
Comment (ivanmerta) [reply src/WorkflowGui/Controller/WorkflowController.php]: Can I check with some tool and see what's not good? Thanks

Comment (dpfaffenbauer) [reply src/WorkflowGui/Controller/WorkflowController.php]: indentation is not good, $placeConfig is indented too much.

Comment (ivanmerta) [reply src/WorkflowGui/Controller/WorkflowController.php]: Ah, damn tabs. Thanks

**Review by dpfaffenbauer — APPROVED** [review]

**Comment by jorisros** [comment]
@ivanmerta thank you

**Pull request merged**